### PR TITLE
Update exercise-tracker.md

### PR DESCRIPTION
**Pull request by Anu790j**
fix: Incomplete argument in test #50135

Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [ ] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [ ] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

### CODE REVIEW

**Comment by Anu790j** [comment]
fix : Incomplete argument in test #50135
I have fixed an open issue.
I am a newbie in Open Source ,if there are mistakes please correct me.
I am open for learning.

**Comment by gikf** [comment]
Hi there, 

Thanks for creating this pull request.

Please ensure you follow the PR checklist and not just tick things off; otherwise, we won't take you seriously. Respect the requirements to earn the respect of your fellow maintainers. You will also save everyone time :)

Since you are new here, please slow down and read our [contributing guidelines](https://contribute.freecodecamp.org), as we see that you may still need to catch up on a few things.

<details>
<summary>Here are some examples (expand)</summary>
 
1. Do not edit files directly on GitHub – while you can, it's not a good idea. 

   Typos and formatting errors can break the tests.

2. Use the correct way to link issues. 

   Add the issue number only in the description of the PR by updating the `XXXXXX`. Do not just add issue numbers everywhere and anywhere you feel like.

4. Keep mentions and review requests to a minimum. 

   We understand you are excited about contributing, and our maintainers will get back to you when they can.

5. Do not work directly off your `main` branch. 

   You can always create a new branch for the changes you are working on. That way, you can sync change to your PR branch as the main repository moves ahead while your PR is waiting in the merge queue.

</details>

Don't worry. You don't have to close this PR. Feel free to ask specific queries on improving your PR here; someone will guide you.

We are happy you are excited to contribute and appreciate you taking the time to help us. Looking forward to more contributions!

Happy Contributing.

File: curriculum/challenges/english/05-back-end-development-and-apis/back-end-development-and-apis-projects/exercise-tracker.md [hunk 1]
```diff
@@ -521,8 +521,8 @@ async(getUserInput) => {
       _id,
       date: new Date().toLocaleDateString("en-US", {
         timeZone: "UTC", weekday: "short", month: "short",
-        day: "2-digit", year: "numeric"
-    };
+        day: "2-digit", year: "numeric" 
+    }).replaceAll(',', '');
```
Comment (gikf) [inline curriculum/challenges/english/05-back-end-development-and-apis/back-end-development-and-apis-projects/exercise-tracker.md:525]:
```suggestion
        day: "2-digit", year: "numeric"
      }).replaceAll(',', '');
    };
```

**Comment by Sboonny** [comment]
@Anu790j, thank you for trying to solve the issue, the issue is solved by another PR, so I am closing this PR.

**Pull request closed without merging**